diff --git a/src/Form/BehanceSettingsForm.php b/src/Form/BehanceSettingsForm.php
index 0744f0a..40426b6 100644
--- a/src/Form/BehanceSettingsForm.php
+++ b/src/Form/BehanceSettingsForm.php
@@ -41,26 +41,26 @@ class BehanceSettingsForm extends ConfigFormBase {
     // API key field.
     $form['api_key'] = array(
       '#type' => 'textfield',
-      '#title' => t('API Key'),
+      '#title' => $this->t('API Key'),
       '#required' => TRUE,
-      '#description' => t('Enter your API key. If you don\'t have one, visit <a target="_blank" href="https://www.behance.net/dev/register">this page</a> and get your Behance API key.'),
+      '#description' => $this->t('Enter your API key. If you don\'t have one, visit <a target="_blank" href="https://www.behance.net/dev/register">this page</a> and get your Behance API key.'),
       '#default_value' => $config->get('api_key'),
     );
 
     // User ID or username field.
     $form['user_id'] = array(
       '#type' => 'textfield',
-      '#title' => t('Username or User ID'),
+      '#title' => $this->t('Username or User ID'),
       '#required' => TRUE,
-      '#description' => t("Enter portfolio owner's ID or username."),
+      '#description' => $this->t("Enter portfolio owner's ID or username."),
       '#default_value' => $config->get('user_id'),
     );
 
     // New tab checkbox.
     $form['new_tab'] = array(
       '#type' => 'checkbox',
-      '#title' => t('Open links in new tab'),
-      '#description' => t('Check if you want Behance links to be opened in the new tab.'),
+      '#title' => $this->t('Open links in new tab'),
+      '#description' => $this->t('Check if you want Behance links to be opened in the new tab.'),
       '#default_value' => $config->get('new_tab'),
     );
 
